docs/releasenotes/3.3.0.rst

```diff
@@ -37,6 +37,14 @@ as in test case files.
 New E0416 ``invalid-setting-in-resource`` rule detects unsupported
 settings in resource files.
 
+New rule that suggests using task-related settings if Task section is used (#522)
+---------------------------------------------------------------------------------
+
+If the file contains ``*** Task ***`` section, it is recommended to use settings like ``Task Setup``,
+``Task Teardown``, ``Task Template`` and ``Task Timeout`` instead of their `Test` variants.
```

Comment:
```suggestion
``Task Teardown``, ``Task Template`` and ``Task Timeout`` instead of their ``Test`` variants.
```

Reply:
Here I added singular backticks on purpose, since "Test" alone is not any part of the code, but it's a word from statements like `Test Teardown`. So here it does not stand as an RF code element, rather as a name for the variants of it.